proto/types.proto

```diff
@@ -36,3 +36,42 @@ message Guid {
   // 128 bits (16 bytes)
   bytes value = 1;
 }
+
+enum HaRole{
+  HA_SCOPE_ROLE_DEAD = 0;
+  HA_SCOPE_ROLE_ACTIVE = 1;
+  HA_SCOPE_ROLE_STANDBY = 2;
+  HA_SCOPE_ROLE_STANDALONE = 3;
+  HA_SCOPE_ROLE_SWITCHING_TO_ACTIVE = 4;
+}
+
+enum HaOwner{
+  OWNER_CONTROLLER = 0;
```

Comment:
Please add `OWNER_CONTROLLER_UNSPECIFIED` similar to above

Reply:
done.